Jupyter/readme.md

```diff
@@ -5,27 +5,27 @@ and [Python installation](https://github.com/QuantConnect/Lean/tree/master/Algor
 
 ### [Windows](https://github.com/QuantConnect/Lean#windows)
 **1. Install Jupyter and its dependencies:**
-   1. Install Jupyter
+   1. Install Jupyter:
 ```
     pip install jupyter
 ```
  2.  Install [QuantConnect Python API](https://pypi.python.org/pypi/quantconnect/0.1)
```

Comment:
Use https://pypi.python.org/pypi/quantconnect instead. It will redirect to the newest version.